contao/core/2014-02-13.yaml

```diff
@@ -2,16 +2,10 @@ title:     PHP object injection vulnerability allows for arbitrary code executio
 link:      https://contao.org/en/news/major-security-hole-found-in-contao.html
 cve:       ~
 branches:
-    2.11.x:
```

Comment:
he is not just adding a new CVE. The title also says `Update the Contao data` (which is often about saying that versions before 2.11 are actually also affected by existing 2.x vulnerabilities)